packages/shared/src/lib/constants.ts

```diff
@@ -40,6 +40,8 @@ export const searchFeedback = 'https://r.daily.dev/search-feedback';
 export const searchDocs = 'https://r.daily.dev/search-docs';
 export const migrateUserToStreaks = 'https://r.daily.dev/streaks';
 
+export const dailyDevApps = 'https://daily.dev/apps';
```

Comment:
This is already a rebrandly link afaik.
If unclear check with nimrod but we decided to only use them in case marketing ever wants to change 👍

Reply:
Updated 🚀